### PR TITLE
cleanup + shutdown of legacy nft.approvals

### PR DESCRIPTION
**Pull request by 0xRobin**
1. moved everything under new `/_sector/nft` directory structure
2. clean op schema.yml files + add column tests for all sub models
3. added a `remove` tag to stop running these models on legacy spark jobs.
4. added a `block_month` column and use that as partitioning to reduce the number of partitions

### CODE REVIEW

File: models/_sector/nft/approvals/legacy/nft_arbitrum_approvals_legacy.sql [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {{ config(
-	tags=['legacy'],
-	
+	tags=['legacy', 'remove'],
```
Comment (couralex6) [inline models/_sector/nft/approvals/legacy/nft_arbitrum_approvals_legacy.sql:2]: What's the `remove` tag here?

Comment (0xRobin) [reply models/_sector/nft/approvals/legacy/nft_arbitrum_approvals_legacy.sql:2]: > What's the `remove` tag here?
This tag will disable the legacy model (both on dbt cloud as in the CI)
It was introduced here: https://github.com/duneanalytics/spellbook/pull/4188

Comment (couralex6) [reply models/_sector/nft/approvals/legacy/nft_arbitrum_approvals_legacy.sql:2]: Ah this was introduced when I was off. Looks great!

**Review by couralex6 — APPROVED** [review]

**Review by jeff-dude — APPROVED** [review]
nice! looking forward to all of us pushing through more `remove` tags

**Pull request merged**